src/LatteTemplateResolver/AbstractClassMethodTemplateResolver.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Efabrica\PHPStanLatte\LatteTemplateResolver;
+
+use Efabrica\PHPStanLatte\Template\Template;
+use PHPStan\BetterReflection\Reflection\ReflectionClass;
+use PHPStan\Node\CollectedDataNode;
+
+abstract class AbstractClassMethodTemplateResolver extends AbstractClassTemplateResolver
+{
+    protected function getClassTemplates(ReflectionClass $reflectionClass, CollectedDataNode $collectedDataNode): array
+    {
+        $globalVariables = $this->getClassGlobalVariables($reflectionClass);
+        $globalComponents = $this->getClassGlobalComponents($reflectionClass);
+
+        $templates = [];
+        foreach ($this->getMethodsMatching($reflectionClass, $this->getClassMethodPattern() . 'i') as $reflectionMethod) {
```

Comment:
why case insensitive?

Reply:
Because method names are case insensitive in PHP